DQMOffline/Trigger/python/DQMOffline_LumiMontiroring_cff.py

```diff
@@ -1,7 +1,7 @@
 import FWCore.ParameterSet.Config as cms
 
 # lumi from scalers
-hltScalersRawToDigi = cms.EDProducer( "ScalersRawToDigi",
+hltScalersRawToDigi4DQM = cms.EDProducer( "ScalersRawToDigi",
```

Comment:
on this, I thought the same when I first introduced it (some months ago)
can you confirm me that `scalersRawToDigi` is *always* run ?
I mean, in prompt reco, in *all* runTheMatrix workflows, etc ?